src/SourceCode.Chasm.Tests/Sha1Tests.cs

```diff
@@ -310,55 +307,7 @@ public static void When_copyto_with_null_buffer()
             var sha1 = new Sha1();
 
             // Action
-            var actual = Assert.Throws<ArgumentNullException>(() => sha1.CopyTo(buffer, 0));
-
-            // Assert
-            Assert.Contains(nameof(buffer), actual.Message);
-        }
-
-        [Trait("Type", "Unit")]
-        [Fact(DisplayName = nameof(When_copyto_with_null_buffer))]
-        public static void When_copyto_with_negative_offset()
```

Comment:
Once converted to use `Span`, these units end up testing System.Buffers itself. Which is needless.